agent/engine/serviceconnect/manager_linux.go

```diff
@@ -115,9 +115,9 @@ type manager struct {
 	// Http path + params to make a drain request of AppNetAgent
 	adminDrainRequest string
 
-	AgentContainerImageName   string
-	AgentContainerTag         string
-	AgentContainerTarballPath string
+	AgentContainerImageName string
```

Comment:
Q. are these variables needed to be exported? should we change them to lower-case?

Reply:
Ya  I can change this to `agentContainerImageName` since it not being exported.